### DISCUSSION TITLE
Hosting options (why we're using Netlify)

### DISCUSSION THREAD

**Discussion by jgaehring** [farmOS/farmOS.org#74] [General]
I'm opening (and will promptly close) this issue just to document some of the decision process in using the Netlify "starter" plan to host this, at least for the time being.

The current version of farmos.org uses GitHub pages, so the question has come up a few times, as with Field Kit (https://github.com/farmOS/farmOS-client/issues/326#issuecomment-602711549), whether we risk additional charges if we exceed Netlify's limits on traffic and build capacity for the starter tier, and how that compares to GitHub pages. So here's the breakdown, as of 2021-11-15:

|              | [GitHub Pages][GHP] | [Netlify Starter Plan][NSP]          |
|--------------|---------------------|--------------------------------------|
| Bandwidth    | 100GB per month     | 100GB /month (then $20 per 100)      |
| Build limits | 10 builds per hour  | 300 minutes /month (then $7 per 500) |

They're pretty evenly matched, at least on those terms, although Netlify comes with a boatload of other free features that GHP doesn't even approach. And as I mentioned in a separate chat, perhaps the greatest advantage of Netlify is they at least give you the _option_ of a higher price tier, if we're ever so fortunate to have that problem, while GitHub only has this advice:

> If your site exceeds these usage quotas, we may not be able to serve your site, or you may receive a polite email from GitHub Support suggesting strategies for reducing your site's impact on our servers, including putting a third-party content distribution network (CDN) in front of your site, making use of other GitHub features such as releases, or moving to a different hosting service that might better fit your needs.

Soooo, yea, to me this is a no brainer, but it's helpful to have the breakdown here for future reference in case it comes up again.

[GHP]: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits
[NSP]: https://www.netlify.com/pricing/#features

**Comment by mstenta** [comment]
Following up on this comment: https://github.com/farmOS/farmOS-community-blog/pull/13#issuecomment-1487646327

I did a little digging and found this: https://www.netlify.com/legal/open-source-policy/ + https://opensource-form.netlify.com/

We may be able to get a free "Pro" plan for the farmOS project with unlimited members! It looks like we already meet all the qualifications. The only additional thing we would need to do is add "This site is powered by Netlify" to farmOS.org.

I'll go ahead and fill out the application form and see what happens... :smile: :shrug:

(Sorry don't mean to resurrect a closed issue, but this seems like the right place to document this for historical records.)

**Reply by jgaehring** [reply]
Converted this to a discussion via labels:

https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#converting-issues-based-on-labels

**Comment by mstenta** [comment]
Update: the farmOS organization has an official Netlify account now! :tada: 

https://app.netlify.com/teams/farmos-org

**Reply by jgaehring** [reply]
Update for the Update: We finally moved farmos.org to the official farmOS Netlify account today! 🎉
